src/table/TableWrapper.jsx

```diff
@@ -49,16 +61,22 @@ export default function TableWrapper(props) {
     return null;
   }
 
+  useEffect(() => {
+    const updateSize = () => setTableWidth(el.clientWidth);
+    window.addEventListener('resize', updateSize);
+    return () => window.removeEventListener('resize', updateSize);
```

Comment:
No need to add another listener since `el.clientWidth` updates on resize. So adding that the dependency array and just call `setTableWidth` should work fine

Reply:
no, not really. I tried the way that you described but it does not work. I belive that the reason is that **const el = useElement();** and **render(el, {})** way to render UI.